docs/naaccr_data.xsd

```diff
@@ -1,21 +1,26 @@
-<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:tns="http://naaccr.org/naaccrdata" targetNamespace="http://naaccr.org/naaccrdata" >
+<xsd:schema xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:tns="http://naaccr.org/naaccrxml" targetNamespace="http://naaccr.org/naaccrxml" >
 
-<xsd:element name="NaaccrData">
+<xsd:simpleType name="recordtypeType">
+	<xsd:restriction base="xsd:string">
+		<xsd:enumeration value="A" />
+		<xsd:enumeration value="M" />
+		<xsd:enumeration value="C" />
+		<xsd:enumeration value="I" />
+	</xsd:restriction>
+</xsd:simpleType>
+
+<xsd:element name="NaaccrDataExchange">
 	<xsd:complexType>
 		<xsd:sequence>
-			<xsd:element ref="tns:Patient" minOccurs="0" maxOccurs="unbounded" />
+			<xsd:element ref="tns:Item" minOccurs="0" maxOccurs="unbounded" />
 			<xsd:any minOccurs="0" maxOccurs="unbounded" />
```

Comment:
I am thinking that the data in the extensions may need to be known before the first Patient is processed, so I favor putting it before Patient but after Item, but it is hard to tell at this stage what would be most appropriate:
```
<NaaccrDataExchange>
     <Item> ...
     <ExtensionAtRootLevel> ...
     <Patient> ...
         <Item> ...
         <ExtensionAtPatientLevel> ...
            <Tumor>
                 <Item> ...
                 <ExtensionAtTumorLevel> ...